app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -588,6 +593,7 @@ class SamplesHeatmapView extends React.Component {
           taxonDetails={this.state.taxonDetails}
           taxonFilterState={this.state.taxonFilterState}
           thresholdFilters={this.state.selectedOptions.thresholdFilters}
+          sortSamples={this.state.selectedOptions.sortSamples}
```

Comment:
Nit: Maybe call this `sampleSort`? Since this contains a sort type like "alpha" or "cluster", `sampleSort` (or `sampleSortType`) sounds slightly better.

Reply:
👍